src/library/sidebarmodel.cpp

```diff
@@ -17,6 +17,9 @@ namespace {
 /// been chosen as a compromise between usability and responsiveness.
 constexpr int kPressedUntilClickedTimeoutMillis = 300;
 
+/// Enables additional debugging output.
+constexpr bool sDebug = false;
```

Comment:
```suggestion
constexpr bool kDebug = false;
```
Our code style is to use the prefix k for const expressions: https://github.com/mixxxdj/mixxx/wiki/Coding-Guidelines#constexpr

Reply:
Thanks, fixed! I was using the existing code as a guideline, which only ever uses `constexpr bool sDebug` (albeit only in two places).